porch/pkg/git/draft.go

```diff
@@ -33,16 +33,17 @@ import (
 )
 
 type gitPackageDraft struct {
-	parent    *gitRepository
-	path      string
-	revision  string
-	lifecycle v1alpha1.PackageRevisionLifecycle // New value of the package revision lifecycle
-	updated   time.Time
-	base      *plumbing.Reference // ref to the base of the package update commit chain (used for conditional push)
-	branch    BranchName          // name of the branch where the changes will be pushed
-	commit    plumbing.Hash       // Current HEAD of the package changes (commit sha)
-	tree      plumbing.Hash       // Cached tree of the package itself, some descendent of commit.Tree()
-	tasks     []v1alpha1.Task
+	parent      *gitRepository
+	path        string
+	revision    string
+	description string
+	lifecycle   v1alpha1.PackageRevisionLifecycle // New value of the package revision lifecycle
```

Comment:
We really should put comments before the fields (in normal go style) so the diffs are more reasonable :-)  But I know it's not strictly related to this PR!

Reply:
Added